Makefile

```diff
@@ -1,20 +1,36 @@
-.PHONY: init build build_docker deploy
+.PHONY: install init build build_docker test deploy start-ls stop-ls
 
 .EXPORT_ALL_VARIABLES:
-AWS_PROFILE = default
 GOPROXY = direct
+NETWORK_NAME="localstack-shared-net"
+
+install: 
+	@which localstack || pip install localstack
+	@which awslocal || pip install awscli-local
 
 init:
-	cd cdk;
+	cd cdk;\
 	npm i
 
 build_docker:
 	docker build -t go-fargate .
 
 deploy: build_docker
 	cd cdk;\
-	cdk deploy --profile ${AWS_PROFILE}
+	cdklocal bootstrap;\
+	cdklocal deploy ---require-approval never
 
-destroy:
```

Comment:
it was removed because it simply was not really needed nor tested with LocalStack, since you can simply restart it, and the non-create CRUD stack operations tend to be problematic, so I didn't really bother with it truth be told.

Reply:
I just checked it again: the stack is destroyed alright, but the resources are still there. Will stick with removing it :)